aten/src/ATen/Overhead.cpp

```diff
@@ -0,0 +1,37 @@
+#include <ATen/ATen.h>
+
+namespace at {
+
+namespace {
+Tensor trivial_empty() {
+  return detail::empty_generic(
+    /*size=*/{1},
+    /*allocator=*/getCPUAllocator(),
+    /*dispatch_key=*/DispatchKey::CPU,
+    /*scalar_type=*/ScalarType::Float,
+    /*device=*/c10::Device(c10::kCPU),
+    /*memory_format_opt=*/c10::MemoryFormat::Contiguous
+  );
+}
+}  // namespace
+
+namespace native {
+
+Tensor _noop_unary(const Tensor& self) {
+  return std::move(trivial_empty());
```

Comment:
The `std::move` here is unnecessary for multiple reasons:
1. The expression is already a temporary, so std::move does nothing
2. Even if it did something, RVO is a thing, so you still don't need the move

Reply:
Done.